script/copy_deduplicate

```diff
@@ -310,15 +309,15 @@ def list_live_tables(client, pool, project_id, only_tables, table_filter):
     ]
 
 
-def main():
+def _main():
```

Comment:
I didn't want to write docstrings for all the functions in scripts, now that pydocstyle working again, so instead I renamed the functions to be private.

This isn't great for `main` functions, but I figured it's fine for now because scripts can't be imported.

Reply:
Yes, that should be fine. Eventually, the logic from the `script/` scripts will be moved to `bigquery_etl`, so those can be renamed back to `main` when that happens.